projects/core/src/anonymous-consents/http-interceptors/anonymous-consents-interceptor.spec.ts

```diff
@@ -298,6 +300,8 @@ describe('AnonymousConsentsInterceptor', () => {
         http.get('/xxx').subscribe();
         tick(DELAY_TIME);
         httpMock.expectOne((req) => req.method === 'GET', 'GET');
+        expect(anonymousConsentsService.getConsents).toHaveBeenCalled();
+        expect(authService.isUserLoggedIn).toHaveBeenCalled();
```

Comment:
Somehow when using fakeAsync and angular mock http, the jasmine complained that this test had no expectations. 

However `httpMock.expectOne((req) => req.method === 'GET', 'GET');` was doing it job (making expectation).
To verify it, I changed the condiditon to "!== 'GET" and then the test failed.

So I'm adding those regular `expect`s just to get rid of the warning 'test has no expectations'. The test has already been and remains correct.